.github/actions/linter_tests/action.yaml

```diff
@@ -44,6 +44,11 @@ runs:
 
         case "$RUNNER_OS" in
           Linux)
+            # Cleanup any existing apt locks
+            sudo killall apt-get || true
+            sudo rm /var/lib/apt/lists/lock || true
```

Comment:
There was a 2-day old apt-get process that still had the lock on one of the runners. This should rarely/never happen, so just clean up as part of the action runner process, because it will fail otherwise anyway